.rubocop.yml

```diff
@@ -16,3 +16,6 @@ AllCops:
 
 Rails/EnvironmentVariableAccess:
   Enabled: false
+
+RSpec/IndexedLet:
```

Comment:
Rubocop started to lint this cop, which I disabled as spec subjects have this pattern.